### PR TITLE
format lists and cleanup, KFP standalone

### PR DESCRIPTION
**Pull request by kbhawkey**
See issue #1622.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1689)

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-1689--competent-brattain-de2d6d.netlify.com/docs/pipelines/installation/standalone-deployment/

**Comment by kbhawkey** [comment]
I am still seeing a few problems with the formatting of lists.

On this page, the main issue is the list (ordered) containing a code block with overflow (scrolling).
If the list is followed immediately by an `<h2>` heading, the break between the two elements is not handled correctly. If the code block has a `lang` attribute set, for example, `shell`, the formatting of the heading is fine but the indentation of the code block is altered.

What is the default `lang` attribute setting for code blocks?
In some places I created an unordered list -- may want to change this back to ordered.

**Comment by joeliedtke** [comment]
The Markdown parser seems to have trouble with code blocks that specify a language. If I recall correctly, for complex docs the following rules work well:

1. Do not include a language in code blocks.
1. For code blocks within a list:
    * Indent the code block by two spaces. 
    * Do not include a blank line between the list and the code block.

For example:

```
1. This is a test.
      ```
      this = is.atest()
      ````
1. This is also a test.
```

**Comment by kbhawkey** [comment]
Looks like the problem with ordered lists and code blocks with overflow (followed by h2 level heading) is not evident on Chrome. The problem (scrolling not working in code block) is resolved if the `margin-bottom` is increased for the ordered list element.

**Comment by kbhawkey** [comment]
- I fiddled with the headings; let me know if this text should be reverted.
- There is duplication between the `deploy to an existing cluster` and `deploy to a new cluster`.
  I Could create a common section accessed from both pathways. This would help clean up the page a bit more and shorten the length of the page.

File: content/docs/pipelines/installation/standalone-deployment.md [hunk 1]
```diff
@@ -206,28 +223,64 @@ bases:
...
 
-Run:
+If you encounter a permission error when installing Kubeflow Pipelines standalone, run:
 
 ```
-kubectl create clusterrolebinding your-binding --clusterrole=cluster-admin --user=[your-user-name]
+kubectl create clusterrolebinding your-binding --clusterrole=cluster-admin --user=<your-user-name>
 ```
+
+## Further reading
```
Comment (joeliedtke) [inline content/docs/pipelines/installation/standalone-deployment.md]: I think these links are important in context. Consider moving these links back into the content of the page.

File: content/docs/pipelines/installation/standalone-deployment.md [hunk 2]
```diff
@@ -1,201 +1,218 @@
...
-platforms. This guide needs updating. See
-<a href="https://github.com/kubeflow/website/issues/1253">issue #1253</a>.</p>
+{{% alert title="Installation options for Kubeflow Pipelines standalone" color="info" %}}
+This guide currently describes how to install Kubeflow Pipelines standalone
+on Google Cloud Platform (GCP). You can also install Kubeflow Pipelines standalone on other
+platforms. This guide needs updating. See [Issue 1253](https://github.com/kubeflow/website/issues/1253).
 {{% /alert %}}
 
-## Common prerequisites
+## Prerequisites
```
Comment (joeliedtke) [inline content/docs/pipelines/installation/standalone-deployment.md]: I would recommend adding a line that describes the purpose of the Prerequisites. 

The Google Developer Docs style guide recommends against [empty headings](https://developers.google.com/style/headings)

File: content/docs/pipelines/installation/standalone-deployment.md (hunk 1 above)
Comment (kbhawkey) [reply content/docs/pipelines/installation/standalone-deployment.md]: OK

**Comment by kbhawkey** [comment]
Hello @joeliedtke .I created a few more adjustments.
- Moved the reference section back in line. Happy to modify the link text if not correct.
- Added content after an h2 heading.
- Pushed the `setting up a new cluster` section to a `Before you get started` section.
- Added a bit of text in `Best practices` as an intro to working with manifests. Though, this section is nice to have, the examples are very close to the previous sections (deploy, upgrade, uninstall).

File: content/docs/pipelines/installation/standalone-deployment.md [hunk 3]
```diff
@@ -1,201 +1,206 @@
...
+      * [gcloud command reference](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create)
 
-    Kubeflow Pipelines applications take a while (~3 minutes) to start.
-    
-    **Note**: The above approach is introduced in Kubeflow Pipelines version 0.2.0. For older versions please run the following instead:
-    ```
-    export PIPELINE_VERSION={{% pipelines/latest-version %}}
-    kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
-    ```
+1. Configure kubectl to talk to your newly created cluster. See the Google Kubernetes Engine (GKE) guide to
```
Comment (joeliedtke) [inline content/docs/pipelines/installation/standalone-deployment.md]: Users will always need to configure kubectl to talk to their cluster. I would recommend moving this to the deploying section or adding a separate before you begin section.

Comment (kbhawkey) [reply content/docs/pipelines/installation/standalone-deployment.md]: Totally agree.

File: content/docs/pipelines/installation/standalone-deployment.md (hunk 2 above)
Comment (kbhawkey) [reply content/docs/pipelines/installation/standalone-deployment.md]: Thanks for the tip.

File: content/docs/pipelines/installation/standalone-deployment.md [hunk 4]
```diff
@@ -1,201 +1,209 @@
...
+
+```
+kubectl delete -k manifests/kustomize/env/gcp
+```
+
+## Best practices for maintaining manifests
 
-## Best practices maintaining custom manifests
+Similar to source code, configuration files belong in source control.
+A repository manages the changes to your
```
Comment (kbhawkey) [inline content/docs/pipelines/installation/standalone-deployment.md]: I wonder if these sentences fit with the rest of the doc.

Comment (joeliedtke) [reply content/docs/pipelines/installation/standalone-deployment.md]: Agreed, this section doesn't seem useful. I would recommend that we split this into a separate issue. It seems like this requires guidance on how to customize a kustomize YAML file, then deploy/update/remove with the custom YAML.

What do you think?

Comment (kbhawkey) [reply content/docs/pipelines/installation/standalone-deployment.md]: Right, I don't think this section (Best practices) adds any new information.
The `Customizing` section outlines (could provide more details) several ways to modify manifest files, but does not explain why you should do this. Could provide some examples of customizing manifest files based on pulling in configuration files from other repositories. Customization (and kustomize) is a larger topic. Also, `kustomize` overlays is mentioned, but not in detail.
Do you want the `Best Practices` removed 😃 ?

Comment (joeliedtke) [reply content/docs/pipelines/installation/standalone-deployment.md]: I think that section could use some work. I'll write this up as a separate issue so we can merge these changes.

**Comment by joeliedtke** [comment]
/lgtm
/approve

Thanks for working on this @kbhawkey!

**Pull request merged**